README.md

```diff
@@ -1,51 +1,3 @@
 # EC-DslDeploy
-Deployment of your DSL code structure to your server.
 
```

Comment:
I guess this README should remain the the old repository only. And probably it is better to place the movement note into the repository description.

Reply:
Great idea. I will do that